packages/workbox-background-sync/src/lib/request-manager.js

```diff
@@ -42,49 +42,58 @@ class RequestManager {
   }
 
   /**
+   * function to play one single request
+   * given its hash
+   * @param {String=} hash
+   * @return {Promise}
```

Comment:
Can you provide some context about what the promise represents? I.e. "Resolves when..., rejects when..."